spec/features/directors_feature_spec.rb

```diff
@@ -0,0 +1,165 @@
+require "rails_helper"
+
+RSpec.describe "Directors", type: :feature do
+
+  before do
+    scorsese = create(:director, name: "Martin Scorsese", dob: "November 17, 1942")
+    departed = create(:movie, title: "The Departed", year: 2006, duration: 151, director: scorsese)
+    goodfellas = create(:movie, title: "Goodfellas", year: 1990, duration: 146, director: scorsese)
+
+    nolan = create(:director, name: "Christopher Nolan", dob: "July 30, 1970")
+    dark_knight = create(:movie, title: "The Dark Knight", year: 2008, duration: 152, director: nolan)
+    inception = create(:movie, title: "Inception", year: 2010, duration: 148, director: nolan)
+
+    leo = create(:actor, name: "Leonardo DiCaprio", dob: "November 11, 1974")
+    create(:character, name: 'Cobb', movie: inception, actor: leo)
+    create(:character, name: 'Billy Costigan', movie: departed, actor: leo)
+
+    jack = create(:actor, name: "Jack Nicholson", dob: "April 22, 1937")
+    create(:character, name: 'Frank Costello', movie: departed, actor: jack)
+
+    bob = create(:actor, name: "Robert De Niro", dob: "August 17, 1943")
+    create(:character, name: 'James Conway', movie: goodfellas, actor: bob)
+  end
+
+  context "index page" do
+    it "displays each director's name and dob", points: 5 do
+      visit "/directors"
+
+      directors = Director.all
+      directors.each do |director|
+        expect(page).to have_content(director.name)
+        expect(page).to have_content(director.dob)
+      end
+    end
+
+    it "displays a functional delete link for each director", points: 5 do
+      visit "/directors"
+
+      count_of_directors = Director.count
+
+      click_link 'Delete', match: :first
+
+      expect(Director.count).to eq(count_of_directors - 1)
+    end
+  end
+
+  context "show page" do
+    it "displays a count of the director's movies", points: 5 do
+      directors = Director.all
+      directors.each do |director|
+        visit "/directors/#{director.id}"
```

Comment:
That would have been in case we wanted to use Factories more rigorously, which it seems we've decided against. So nevermind :)